### DISCUSSION TITLE
Error while updating Sonarr v4

### DISCUSSION THREAD

**Discussion by nfribeiro** [tteck/Proxmox#2304] [Help Needed]
In the pass I had no problem in updating Sonarr. Today I discovered there's a new version but when I run the command and after the prompt tu update or not, i get this message and the script stops.
/dev/fd/63: line 69: SPINNER_PID: unbound variable

Please help.

**Comment by tteck** [comment]
SSH or Sonarr Console
https://github.com/tteck/Proxmox/discussions/1738#discussioncomment-8005107

**Reply by nfribeiro** [reply]
Thanks but already did that and it doesn't worked, same error.

**Comment by LKStu** [comment]
I'm hanging on here. I had the same mistake last week and now when I try to update via the console with command '_update_' I get "_No Sonarr Installation Found_".

I am on Version 4.0.0.738

Following that link from above I guess I have to run these commands?
```
systemctl stop sonarr.service
cd /tmp 
wget -O SonarrV4.tar.gz 'https://services.sonarr.tv/v1/download/develop/latest?version=4&os=linux'
tar -xzf SonarrV4.tar.gz
cp -r Sonarr/* /usr/lib/sonarr/bin
rm -rf Sonarr SonarrV4.tar.gz
systemctl start sonarr.service
```

**Reply by tteck** [reply]
## 2024-01-03

### Changed

- **Sonarr LXC**
  - Breaking Change
  - Complete recode
  - https://github.com/tteck/Proxmox/discussions/1738#discussioncomment-8005107

**Reply by tteck** [reply]
Start reading here https://github.com/tteck/Proxmox/discussions/1738#discussioncomment-8005867

**Reply by LKStu** [reply]
Thx. Dont know if this is the default correct way but I just created a new container with your script and restored from a backup I made on the old container. Thanks for all your work!
